mini-wallet.md

```diff
@@ -113,25 +109,43 @@ Options:
+The `diem.testing.suites` package contains the Diem MiniWallet Test Suite and includes the pytest file `conftest.py`.
+
+Launch a pytest runner to run tests using:
+
+`dmv test`
+
+The `conftest.py` starts a stub MiniWallet as a counterparty service for testing payment with the target server specified by the --target option here.
+
+
+### Work with your local test network
 
+The Diem MiniWallet server connects to the Diem testnet by default. However, you can use the MiniWallet to test your wallet application in your own local test network as well.
 
-### Working with a local testnet
+To run your local test network:
+1. Download and install Docker and Docker Compose. 
+2. Download the Diem testnet docker compose configuration file.
+3. Run the following command: `docker-compose -f <path to your validator-testnet/docker-compose.yaml file> up --detach`
+	* The Faucet service will now be available at http://127.0.0.1:8000.
+	* A JSON-RPC server will now be available at http://127.0.0.1:8080.
+4. Test your wallet application with your local test network using:
+`dmw test --jsonrpc http://127.0.0.1:8080 --faucet http://127.0.0.1:8000 --target http://localhost:9999`
 
-1. [Download](https://docs.docker.com/get-docker/) and install Docker and Docker Compose (comes with Docker for Mac and Windows).
-2. Download Diem testnet docker compose config: https://github.com/diem/diem/blob/master/docker/compose/validator-testnet/docker-compose.yaml
-3. Run `docker-compose -f <path to your validator-testnet/docker-compose.yaml file> up --detach`
-   * A Faucet will be available at http://127.0.0.1:8000
-   * A JSON-RPC server will be available at http://127.0.0.1:8080
-4. Test your application with a local testnet: `dmw test --jsonrpc http://127.0.0.1:8080 --faucet http://127.0.0.1:8000 --target http://localhost:9999`
 
-### Test Off-chain API
+### Test Off-chain APIs
 
-As the test counterparty wallet application server is started locally, you need make sure your wallet application's off-chain API can access the stub server using it's base_url: `http://localhost:<port>`.
-If your wallet application is not running locally, you will need to setup a tunnel for your wallet application to access the stub server.
+Off-chain protocols are APIs and payload specifications that support compliance and scalability on the Diem Payment Network (DPN). This is executed between pairs of Regulated VASPs, such as wallets, exchanges, or designated dealers, and allows Regulated VASPs to privately exchange payment information before, while, or after settling it on the Diem Blockchain.
+ 
+These off-chain APIs also provide a means to comply with the Travel Rule and negotiate one-time identifiers for on-chain transactions, reducing transaction linkability. To establish a connection, VASPs will look up their counterparty’s on-chain account containing a base URL for their off-chain service. 
+ 
+#### Access the stub server
```

Comment:
this is not correct title for the content.

my suggestion is keep original paragraph format.
these are more like notes for the case when testing off-chain api failed, what could be problem (the target wallet application can't access stub wallet application offchain api, because the stub wallet application is launched at local.)